src/Lykke.Logs/LastResortLogFactory.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Concurrent;
+using Common.Log;
+using JetBrains.Annotations;
+using Lykke.Common.Log;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+
+namespace Lykke.Logs
+{
+    /// <summary>
+    /// Last resort log factory. It logs only to the console and can't be filtered out. 
+    /// Could be used to log events of logging system, when more complex loggers faced with failure
+    /// </summary>
+    [PublicAPI]
+    public sealed class LastResortLogFactory : ILogFactory
+    {
+        /// <summary>
+        /// Instance of the last resort log factory
+        /// </summary>
+        [NotNull]
+        public static ILogFactory Instance { get; } = new LastResortLogFactory();
+
+        private readonly ConcurrentDictionary<(Type Type, string Suffix), LastResortLog> _logs;
+
+        private LastResortLogFactory()
+        {
+            _logs = new ConcurrentDictionary<(Type, string), LastResortLog>();
+        }
+
+        public ILog CreateLog<TComponent>(TComponent component, string componentNameSuffix)
+        {
+            return _logs.GetOrAdd((component.GetType(), componentNameSuffix), (key) =>
```

Comment:
Check that component.GetType() != componentNameSuffix, if values are the same then it must be processed as CreateLog<TComponent>(TComponent component) call